pkg/asset/cluster/alibabacloud/alibabacloud.go

```diff
@@ -0,0 +1,15 @@
+package alibabacloud
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/alibabacloud"
+)
+
+// Metadata converts an install configuration to Alibaba Cloud metadata.
+func Metadata(config *types.InstallConfig) *alibabacloud.Metadata {
+	return &alibabacloud.Metadata{
+		Region:          config.Platform.AlibabaCloud.Region,
+		ResourceGroupID: config.Platform.AlibabaCloud.ResourceGroupID,
+		ClusterDomain:   config.ClusterDomain(),
```

Comment:
This is unused.

Reply:
'ClusterDomain' is used in [here](https://github.com/openshift/installer/blob/61360645cc340b28a41efd50e8ac71fde28346ca/pkg/destroy/alibabacloud/alibabacloud.go#L142)